.github/workflows/build.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       matrix:
         # Pin ubuntu to ensure mono is installed
         # Pin macos to get the version of Xcode that we need: https://github.com/actions/runner-images/issues/10703
-        os: [ubuntu-22.04, windows-latest, macos-15]
+        os: [ubuntu-22.04, windows-latest, windows-11-arm, macos-15]
```

Comment:
via `./.github/actions/environment/action.yml`

_Eclipse Temurin_ Java versions `11` (LTS) and `17` (LTS) are not available for _Windows aarch64_.
Available are `21` (LTS) and `23`.

What would upgrading or rolling forward incur?

Reply:
Not sure tbqh. This isn't critical so we can look at it later

Reply:
Trying through: [`283f080` (#4187)](https://github.com/getsentry/sentry-dotnet/pull/4187/commits/283f080a9462f697bf2c8ad7980522d2d62c3405)